src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -441,9 +494,67 @@ item.appliedenergistics2.ToolBiometricCard.name=Biometriekarte
 item.appliedenergistics2.ToolDebugCard.name=Dev.Debug-Karte
 item.appliedenergistics2.ToolReplicatorCard.name=Dev.Replikator-Karte
 
+// Commands
 commands.ae2.usage=Befehle verfügbar in Applied Energistics 2 - Verwende '/ae2 list' für eine Liste und '/ae2 help _____' für Hilfe zu einem Befehl.
 commands.ae2.permissions=Du hast nicht die erforderlichen Berechtigungen, um diesen Befehl auszuführen.
 commands.ae2.ChunkLogger=Schalted das Berichten von Chunk-Loading und Chunk-Unloading im Server-log um. ( OP )
-commands.ae2.ChunkLoggerOn=Chunk Logging ist nun aktiviert
-commands.ae2.ChunkLoggerOff=Chunk Logging is nun deaktiviert
+commands.ae2.ChunkLoggerOn=Chunk Logging ist aktiviert
+commands.ae2.ChunkLoggerOff=Chunk Logging ist deaktiviert
 commands.ae2.Supporters=Zeigt eine Liste aller AE2-Unterstützer
+
+// Achievements
+achievement.ae2.Compass=Meteoritenjäger
+achievement.ae2.Compass.desc=Fertige einen Meteoritenkompass
+achievement.ae2.Presses=Unbekannte Technologie
+achievement.ae2.Presses.desc=Finde eine Prozessorenpresse
+achievement.ae2.SpatialIO=Raumkoordinaten
+achievement.ae2.SpatialIO.desc=Fertige einen IO-Raumport
+achievement.ae2.SpatialIOExplorer=To boldly go
```

Comment:
Nope, it's intentional, since I wasn't sure how translate the phrase. I have to admit, I've never seen Star Wars and I don't know how they've translated it in the German version.